memgpt/local_llm/json_parser.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+
+def extract_first_json(string):
+    """Handles the case of two JSON objects back-to-back"""
```

Comment:
This just counts opening vs closing brackets to try and pull the first JSON object out of a potential "run-on JSON object"